libkbfs/key_manager_test.go

```diff
@@ -2019,7 +2019,7 @@ func testKeyManagerRekeyAddDeviceWithPromptViaFolderAccess(t *testing.T, ver Met
 		defer ops.mdWriterLock.Unlock(lState)
 
 		// Now cause a paper prompt unlock via a folder access
-		errCh := make(chan error)
+		errCh := make(chan error, 1)
```

Comment:
What's the theory on why this could fix the hang?  It seems like the call to `GetRootNodeForTest` will always need to call into the crypto object before returning an error, and so it should block on that `c` receive, after which the test immediately drains `errCh`.  So I don't quite see how buffering would help...